### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by rooftopcellist**
##### SUMMARY

Updates docs for OAuth2

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

 - API
 

##### ADDITIONAL INFORMATION
Cleans up docs.

### CODE REVIEW

File: docs/auth/oauth.md [hunk 1]
```diff
@@ -99,42 +88,45 @@ Individual tokens will be accessible via their primary keys:
...
+    "token": "*************",
+    "refresh_token": "**************",
+    "application": 1,
+    "expires": "2018-02-24T00:39:32.618279Z",
     "scope": "read"
-}
+},
 ```
-For an OAuth token, the only fully mutable field is `scope`. The `application` field is *immutable
+For an OAuth 2 token, the only fully mutable field is `scope`. The `application` field is *immutable
```
Comment (AlanCoding) [inline docs/auth/oauth.md]: See https://github.com/ansible/awx/issues/1376 for bug on immutability.

If this is expected behavior, then bug stays and docs get 👍

File: docs/auth/oauth.md [hunk 2]
```diff
@@ -99,42 +88,45 @@ Individual tokens will be accessible via their primary keys:
...
 on update*, and all other fields are totally immutable, and will be auto-populated during creation:
```
Comment (AlanCoding) [inline docs/auth/oauth.md]: > and all other fields are totally immutable

That's not right, `description` field can be modified, and I see no reason it shouldn't be modifiable. I would change the docs in this case.

File: docs/auth/oauth.md [hunk 3]
```diff
@@ -99,42 +88,45 @@ Individual tokens will be accessible via their primary keys:
...
 `user` field will be the `user` field of related application; `expires` will be generated according
 to Tower configuration setting `OAUTH2_PROVIDER`; `token` and `refresh_token` will be auto-generated 
-to be non-crashing random strings.
+to be non-crashing random strings.  Both application tokens and personal access tokens will be shown
```
Comment (AlanCoding) [inline docs/auth/oauth.md]: What does non-crashing mean? Does it just mean "unique"?

File: docs/auth/oauth.md [hunk 4]
```diff
@@ -158,39 +150,40 @@ According to OAuth 2 specification, users should be able to acquire, revoke and
 token. In AWX the equivalent, and the easiest, way of doing that is creating a token, deleting
 a token, and deleting a token quickly followed by creating a new one.
 
-On the other hand, the specification also provides standard ways of doing those. RFC 6749 elaborates
-on those topics, but in summary, an OAuth token is officially acquired via authorization using
+The specification also provides standard ways of doing this though. RFC 6749 elaborates
```
Comment (AlanCoding) [inline docs/auth/oauth.md]: This sentence is unclear, maybe just delete the word "through"

**Comment by AlanCoding** [comment]
In the session docs it says:

> According to OAuth 2 specification, users should be able to acquire, revoke and refresh an access token. In AWX the equivalent, and the easiest, way of doing that is creating a token, deleting a token, and deleting a token quickly followed by creating a new one.

That last one "deleting a token quickly followed by creating a new one" sounds fishy. I would think that if the user deleted their token and and quickly tried to create a new one, that it wouldn't work, right?

File: docs/auth/oauth.md (hunk 1 above)
Comment (rooftopcellist) [reply docs/auth/oauth.md]: This is fixed here: https://github.com/ansible/awx/pull/1378

**Comment by rooftopcellist** [comment]
@AlanCoding Re: token deletion, It will work to delete a token and make one directly afterwards (by the same user).  A user can make multiple tokens with multiple scopes.

**Review by AlanCoding — APPROVED** [review]

**Pull request merged**